Build failure because of Math.clamp

**Describe the bug**
Lightzone fail to build because of Math.clamp()

**To Reproduce**
Steps to reproduce the behavior:
1. build Lightzone
2. 
**Expected behavior**
Builds

**Screenshots**
```
    [javac] Compiling 1038 source files to /run/build/lightzone/lightcrafts/build/classes/java/main
    [javac] /run/build/lightzone/lightcrafts/src/main/java/com/lightcrafts/model/ImageEditor/WhitePointOperationImpl.java:152: error: cannot find symbol
    [javac]                     table[0][i] = (short) (0xffff & (int) Math.clamp(i + 0xff * interpolator.interpolate(i / (double) 0xffff, redCurve), 0, 0xffff));
    [javac]                                                               ^
    [javac]   symbol:   method clamp(double,int,int)
    [javac]   location: class Math
    [javac] /run/build/lightzone/lightcrafts/src/main/java/com/lightcrafts/model/ImageEditor/WhitePointOperationImpl.java:156: error: cannot find symbol
    [javac]                     table[1][i] = (short) (0xffff & (int) Math.clamp(i + 0xff * interpolator.interpolate(i / (double) 0xffff, greenCurve), 0, 0xffff));
    [javac]                                                               ^
    [javac]   symbol:   method clamp(double,int,int)
    [javac]   location: class Math
    [javac] /run/build/lightzone/lightcrafts/src/main/java/com/lightcrafts/model/ImageEditor/WhitePointOperationImpl.java:160: error: cannot find symbol
    [javac]                     table[2][i] = (short) (0xffff & (int) Math.clamp(i + 0xff * interpolator.interpolate(i / (double) 0xffff, blueCurve), 0, 0xffff));
    [javac]                                                               ^
    [javac]   symbol:   method clamp(double,int,int)
    [javac]   location: class Math
    [javac] Note: Some input files use or override a deprecated API.
    [javac] Note: Recompile with -Xlint:deprecation for details.
    [javac] Note: Some input files use unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.
    [javac] 3 errors
```

**Desktop (please complete the following information):**
 - Version: 5.0.0
 - OS: Linux (Flatpak)
 - Java: jdk-21.0.11+10

**Additional context**
Add any other context about the problem here.

--- a/lightcrafts/build.xml
+++ b/lightcrafts/build.xml
@@ -98,7 +98,7 @@
     <mkdir dir="build/classes/java/main"/>
     <mkdir dir="javah"/>
     <javac srcdir="src/main/java" destdir="build/classes/java/main" includeantruntime="false" debug="true"
-      release="17" nativeheaderdir="javah" classpathref="compile.classpath">
+      release="21" nativeheaderdir="javah" classpathref="compile.classpath">
       <compilerarg value="-proc:full" />
     </javac>
   </target>

--- a/linux/build.xml
+++ b/linux/build.xml
@@ -46,7 +46,7 @@
     <mkdir dir="build/classes/java/main"/>
     <mkdir dir="javah"/>
     <javac destdir="build/classes/java/main" includeantruntime="false" debug="true"
-           release="17" nativeheaderdir="javah">
+           release="21" nativeheaderdir="javah">
       <src path="src/main/java"/>
       <compilerarg value="-proc:full" />
       <classpath>

--- a/macosx/build.xml
+++ b/macosx/build.xml
@@ -41,7 +41,7 @@
     <mkdir dir="build"/>
     <mkdir dir="javah"/>
     <javac destdir="build" includeantruntime="false" debug="true"
-           release="17" nativeheaderdir="javah">
+           release="21" nativeheaderdir="javah">
       <src path="src/main/java"/>
       <compilerarg value="-XDignore.symbol.file -proc:full"/>
       <classpath>

--- a/windows/build.xml
+++ b/windows/build.xml
@@ -60,7 +60,7 @@
     <mkdir dir="build"/>
     <mkdir dir="javah"/>
     <javac destdir="build" debug="true"
-           release="17" nativeheaderdir="javah">
+           release="21" nativeheaderdir="javah">
       <src path="src/main/java"/>
       <compilerarg value="-proc:full" />
       <classpath>

